lib/senders/templates/verify_secondary.txt

```diff
@@ -0,0 +1 @@
+TODO Add template
```

Comment:
Are these templates coming in one of the subsequent PRs or this one? Just making sure it's not something that'd been forgotten about (and it seems weird adding a TODO in one PR to be picked up in a following PR).